doc/man/task-sync.5.in

```diff
@@ -144,6 +144,9 @@ Users are identified by a client ID, and users with different client IDs are
 entirely independent.  Task data is encrypted by Taskwarrior, and the sync
 server never sees un-encrypted data.
 
+The server is developed in
+https://github.com/GothenburgBitFactory/taskchampion-sync-server.
+
```

Comment:
Would it make sense to remove the rest of the section -- from this point to the end? It seems a bit odd to document the command line interface here now that it's a separate project.

Reply:
That makes sense.